### ISSUE TITLE
Weird install location for libcmark.a

### ISSUE THREAD

**Issue by t6** [commonmark/cmark#238]
cmark 0.28.1 now installs `libcmark.a` into `${CMAKE_INSTALL_PREFIX}/include` which seems like an odd choice to me. Is this intentional? 

Ran into this while trying to update the FreeBSD port. I currently patch it like this:
```diff
--- src/CMakeLists.txt.bak	2017-10-11 06:13:55.000000000 +0200
+++ src/CMakeLists.txt	2017-10-12 12:33:38.920390000 +0200
@@ -130,7 +130,7 @@
   EXPORT cmark
   RUNTIME DESTINATION bin
   LIBRARY DESTINATION ${libdir}
-  ARCHIVE DESTINATION include
+  ARCHIVE DESTINATION ${libdir}
   )
 
 if(CMARK_SHARED OR CMARK_STATIC)
```

**Comment by jgm** [comment]
Oops, I'm not sure how that happened!  Sorry about that.
It should now be fixed in the 0.28.2 release.

**Comment by t6** [comment]
Thank you  for fixing this so quickly and putting out a new release.

I just committed the update to the FreeBSD ports tree: https://svnweb.freebsd.org/changeset/ports/451911
